manim/scene/scene.py

```diff
@@ -99,6 +99,12 @@ def render(self):
             self.construct()
         except EndSceneEarlyException:
             pass
+
+        if (self.renderer.num_plays == 0 and not config['save_last_frame']):
+            config['write_to_movie'] = False
+            config['save_last_frame'] = True
+            self.renderer.file_writer.init_save_image(self.__class__.__name__)
```

Comment:
Is it not an option to simply call `init_output_directories` instead of this custom function for images only?